src/worker.py

```
@@ -2708,11 +2708,67 @@ async def _dispatch(request, env):
+        try:
+            options = to_js(
+                {
+                    "method": "POST",
+                    "headers": {"Content-Type": "application/json"},
+                    "body": json.dumps({"text": message}),
+                },
+                dict_converter=js.Object.fromEntries,
+            )
+            await js.fetch(slack_url, options)
+        except Exception:
+            pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)